source/partials/next_release/_release.md.erb

```diff
@@ -1,41 +1,89 @@
-    <h5>Authentication plugins</h5>
-    <ul>
-      <li><p><%= link_to('Google OAuth Login', 'https://github.com/gocd-contrib/gocd-oauth-login') %></p></li>
-      <li><p><%= link_to('GitHub OAuth Login', 'https://github.com/gocd-contrib/gocd-oauth-login') %></p></li>
-      <li><p><%= link_to('GitLab(CE) OAuth Login', 'https://github.com/gocd-contrib/gocd-oauth-login') %></p></li>
-      <li><p><%= link_to('TLS Client Certificates Login', 'https://github.com/cnorthwood/gocd-tls-auth') %></p></li>
       <li><p><%= link_to('Strong Auth Plugin', 'https://github.com/danielsomerfield/go-strong-auth-plugin#go-strong-auth-plugin') %></p></li>
       <li><p><%= link_to('Guest Login', 'https://github.com/gocd-contrib/gocd_auth_plugin_guest_user') %></p></li>
     </ul>
+* Support for phusion based docker [server](https://hub.docker.com/r/gocd/gocd-server-deprecated/) and [agent](https://hub.docker.com/r/gocd/gocd-agent-deprecated/) images have been removed this release. We provide support for the docker images for GoCD agents across the following operating systems - Alpine [3.5](https://hub.docker.com/r/gocd/gocd-agent-alpine-3.5/), CentOS [6](https://hub.docker.com/r/gocd/gocd-agent-centos-6/) & [7](https://hub.docker.com/r/gocd/gocd-agent-centos-7/) , Debian [7](https://hub.docker.com/r/gocd/gocd-agent-debian-7/) & [8](https://hub.docker.com/r/gocd/gocd-agent-debian-8/), Ubuntu [12.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-12.04/), [14.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-14.04/) & [16.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-16.04/)
+* <%= link_to_commit '8571bc13a55853ecff3f19e59f312b2e8b7a1e60', 'Removes garage controller and associated services' %>
+
 
- * As of 17.8, the phusion based docker [server](https://hub.docker.com/r/gocd/gocd-server-deprecated/) and [agent](https://hub.docker.com/r/gocd/gocd-agent-deprecated/) images are deprecated in favour of the new docker [server](https://hub.docker.com/r/gocd/gocd-server/) and agent images. Support for the phusion images will be removed in 17.11.0.
- We provide support for the docker images for GoCD agents across the following operating systems - Centos [6](https://hub.docker.com/r/gocd/gocd-agent-centos-6/) & [7](https://hub.docker.com/r/gocd/gocd-agent-centos-7/) , Debian [7](https://hub.docker.com/r/gocd/gocd-agent-debian-7/) & [8](https://hub.docker.com/r/gocd/gocd-agent-debian-8/), Ubuntu [12.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-12.04/), [14.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-14.04/) & [16.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-16.04/)
+<h4>Other Changes</h4>
+* <%= link_to_commit 'f501efc5a00519461fdf7d93081f83fcca7df2b9', 'Move over to logback', 3814 %>
```

Comment:
Add more detail or link to  issues and PRs that talk about it. There is a breaking change with respect to the log locations of go-server.out.log, and with respect to agent logs -
 https://github.com/gocd/gocd/issues/3588 , https://github.com/gocd/gocd/issues/3557, https://github.com/gocd/gocd/issues/3403